fadin/init.py

```diff
@@ -0,0 +1,259 @@
+import torch
+import matplotlib.pyplot as plt
```

Comment:
Is this needed?

Reply:
I think so because both imports are used in the file. Am I missing something?